pkg/fanal/handler/misconf/misconf.go

```diff
@@ -194,7 +193,6 @@ func newMisconfPostHandler(artifactOpt artifact.Option) (handler.PostHandler, er
 			types.Dockerfile:     dfscanner.NewScanner(opts...),
 			types.Kubernetes:     k8sscanner.NewScanner(opts...),
 			types.Helm:           helm.New(helmOpts...),
-			types.Rbac:           rbac.NewScanner(opts...),
```

Comment:
Seems like this const is not used anywhere. Can we remove it?
https://github.com/aquasecurity/trivy/blob/c575d6f7de4969aa82605c0186166e741276de15/pkg/fanal/types/const.go#L46